src/bundle/Resources/translations/forms.en.xliff

```diff
@@ -161,861 +161,6 @@
         <target>All modules / All functions</target>
         <note>key: role.policy.all_modules_all_functions</note>
       </trans-unit>
-      <trans-unit id="517e3666a759ee26e02fcc468446ef39b3ae9bf0" resname="role.policy.catalog">
```

Comment:
those should not be removed if we have no replacement in other packages yet

Reply:
I think we will coordinate merging this PR together with other PRs adding translations back in correct packages

Reply:
I was thinking that it is leftover because PR is about adding deprecations and has no link to other PRs. Just a friendly reminder to don't merge it as is.